woocommerce-services.php

```diff
@@ -1523,7 +1523,7 @@ public function edit_orders_page_actions() {
 			// Add the WCS&T to WCShipping migratio notice, creating a button to update.
 			$settings_store = $this->get_service_settings_store();
 			$schemas_store  = $this->get_service_schemas_store();
```

Comment:
[Optional]
We can also remove `$schemas_store = $this->get_service_schemas_store();`, doesn't look like it is in used anymore.

Reply:
Good call, totally forgot to remove that var!